create_aio_app/utils/config.py

```diff
@@ -40,11 +41,6 @@ def parse_arguments() -> dict:
         action='store_true',
         help='added redis settings to the generating project'
     )
-    parser.add_argument(
```

Comment:
This this whole module still needed?

Reply:
Yes, i think that `create-aio-app test_project ` looks better than output  from cookiecutter `project_name [app]: test_project`.

Reply:
It looks like you're shadowing built-in CLI features with this: https://cookiecutter.readthedocs.io/en/latest/_modules/cookiecutter/cli.html

Also, since cookiecutter uses click you'd better do the same as it's already pulled in as a dependency.

Reply:
Their docs already have this example that user can pre-fill context within CLI. Don't reinvent the wheel:
> Inject extra context with command-line arguments:
```bash
$ cookiecutter --no-input gh:msabramo/cookiecutter-supervisor program_name=foobar startsecs=10
```

In our case it'd be usable as `cookiecutter gh:aio-libs/create-aio-app project_name=blah use_postgres=n use_redis=y`. So why reinvent the wheel?

Anyway, I think own CLI should belong to a separate place from the template thing.

Reply:
u can use cookiecutter with `gh:aio-libs/create-aio-app/` and i will add link to tempalte into cookiecutter docs https://github.com/audreyr/cookiecutter#python
but i make a easy way to set up.

```
create-aio-app new_project
```
and ready

Reply:
+ in future i want to implement some like this.

```
create-aio-app new_project
```
and after some time in existing project

```
cd new_project
create-aio-app --add nginx
```

or 
```
create-aio-app --add heroku
```

So i think that i need own cli for that

Reply:
\+